styleguide/source/assets/scss/02-molecules/_hero-evergreen.scss

```diff
@@ -0,0 +1,88 @@
+.hero-evergreen {
+  display: flex;
+  flex-flow: column wrap;
+
+  @include breakpoint($bp-med) {
+    flex-flow: row nowrap;
+  }
+
+  &__mobile-image {
+    display: block;
+
+    @include breakpoint($bp-small) {
+      display: none;
+    }
+  }
+
+  &__aside {
+    @include gutter-all(padding, $gutter / 2, $gutter-mobile / 2);
+    background: $purple;
+    flex-basis: 33.33%;
+    color: $white;
+
+    &--full-width {
+      flex-basis: 100%;
+    }
+  }
+
+  &__background {
+    flex-basis: 66.66%;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    background-size: cover;
+
+    @include breakpoint($bp-small max-width) {
+      background: none !important;
+    }
+
+    .hero-evergreen__buttons {
+      display: flex;
+      background: $white;
+      padding-top: $gutter;
+      padding-bottom: $gutter;
+      flex-flow: column wrap;
+      justify-content: space-evenly;
+      min-height: 200px;
+      padding-top: 15px;
+      align-self: auto;
+
+      //Fix for overriding jquery widget
+      .ama__button {
```

Comment:
If your already adding styling for buttons can you match the padding from the zeplins?

Reply:
Done. I also reduced the font weight of the buttons per the zeplins.